independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/ListExtensionsCommandHandler.java

```diff
@@ -34,15 +37,23 @@
     private static final String NAME_FORMAT = "%-50s";
     private static final String ORIGINS_FORMAT = "%-50s %-60s %s";
 
+    private static final String MORE_INFO_HINT = "To get more information, append `%s` to your command line.";
+    private static final String FILTER_HINT = "To list only extensions from specific category, append " +
+            "`%s` to your command line.";
+    private static final String ADD_EXTENSION_HINT = "Add an extension to your project by adding the dependency to your " +
+            "%s or use `%s`";
+
```

Comment:
I would move these to ListExtensions, rather than having them as private attributes on the handler